src-docs/src/views/progress/progress_example.js

```diff
@@ -137,26 +146,68 @@ export const ProgressExample = {
       demo: <ProgressFixed />,
     },
     {
-      title: 'Progress has a range of sizes and colors',
+      title: 'Sizes',
       source: [
         {
           type: GuideSectionTypes.JS,
-          code: progressSizeColorSource,
+          code: progressSizesSource,
         },
         {
           type: GuideSectionTypes.HTML,
-          code: progressSizeColorHtml,
+          code: progressSizesHtml,
         },
       ],
       text: (
         <p>
-          Both <EuiCode>size</EuiCode> and <EuiCode>color</EuiCode> can be
-          provided as props. These values will work on both determinate and
+          You can adjust the <EuiCode>size</EuiCode> of both determinate and
           indeterminate progress bars.
         </p>
       ),
-      demo: <ProgressSizeColor />,
-      snippet: progressSizeColorSnippet,
+      demo: <ProgressSizes />,
+      snippet: progressSizesSnippet,
+    },
+    {
+      title: 'Colors',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: progressColorsSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: progressColorsHtml,
+        },
+      ],
+      text: (
+        <div>
+          <p>
+            <strong>EuiProgress</strong> supports a few options for{' '}
+            <EuiCode>color</EuiCode>. You can pass any value from our basic
+            color set or from our visualization palette (<EuiCode>vis0</EuiCode>{' '}
+            through <EuiCode>vis9</EuiCode>). To learn more about color usage,
+            go to the <Link to="/guidelines/colors">Color guidelines</Link>{' '}
+            page.
+          </p>
+          <p>
+            Additionally, you can pass any valid color string like a hex value
+            or named color.
+          </p>
+          <EuiCallOut
+            title="Note about using custom colors"
+            iconType="accessibility"
+            color="warning">
+            <p>
+              Usually, we calculate a high contrast color for{' '}
+              <EuiCode>valueText</EuiCode>
```

Comment:
Missing a space here
```suggestion
              <EuiCode>valueText</EuiCode>{' '}
```